src/ansiblelint/schemas/changelog.json

```diff
@@ -70,15 +230,14 @@
     "semver": {
       "pattern": "\\d+.\\d+.\\d+.*",
```

Comment:
See "Is there a suggested regular expression (RegEx) to check a SemVer string?" on https://semver.org/

Reply:
JSON specification does only allow very limited set of regex. https://json-schema.org/understanding-json-schema/reference/regular_expressions.html -- very limited!